packages/www/components/Dashboard/AssetsTable/CreateAssetDialog.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Box,
+  Button,
+  Flex,
+  AlertDialog,
+  AlertDialogTitle,
+  AlertDialogContent,
+  AlertDialogCancel,
+  AlertDialogDescription,
+  TextField,
+  Heading,
+  Text,
+  Label,
+} from "@livepeer.com/design-system";
+import { useState } from "react";
+import Spinner from "components/Dashboard/Spinner";
+
+const CreateAssetDialog = ({
+  isOpen,
+  onOpenChange,
+  onCreate,
+}: {
+  isOpen: boolean;
+  onOpenChange: (isOpen: boolean) => void;
+  onCreate: ({ name, url }: { name: string; url: string }) => Promise<void>;
+}) => {
+  const [creating, setCreating] = useState(false);
+  const [assetName, setAssetName] = useState("");
+  const [assetUrl, setAssetUrl] = useState("");
+
+  return (
+    <AlertDialog open={isOpen} onOpenChange={onOpenChange}>
+      <AlertDialogContent css={{ maxWidth: 450, px: "$5", pt: "$4", pb: "$4" }}>
+        <AlertDialogTitle as={Heading} size="1">
+          Create a new asset
+        </AlertDialogTitle>
+
+        <Box
+          css={{ mt: "$3" }}
+          as="form"
+          onSubmit={async (e) => {
+            e.preventDefault();
+            if (creating) {
+              return;
+            }
+            setCreating(true);
+            try {
+              await onCreate({ name: assetName, url: assetUrl });
+            } catch (error) {
+              console.error(error);
+            } finally {
+              setCreating(false);
+            }
+          }}>
+          <Flex direction="column" gap="2">
+            <Label htmlFor="assetName">Asset name</Label>
+            <TextField
+              required
+              size="2"
+              type="text"
+              id="assetName"
+              autoFocus={true}
+              value={assetName}
+              onChange={(e) => setAssetName(e.target.value)}
+              placeholder="e.g. My video asset"
+            />
+          </Flex>
+          <AlertDialogDescription
+            as={Text}
+            size="3"
+            variant="gray"
+            css={{ mt: "$1", fontSize: "$2", mb: "$4" }}>
+            The name of the `Asset` containing a custom human-readable title.
+          </AlertDialogDescription>
+          <Flex direction="column" gap="2">
+            <Label htmlFor="ingestUrl">Asset URL</Label>
+            <TextField
+              required
+              autoFocus
+              size="2"
+              type="url"
+              pattern="^(http|https?)://.+"
+              id="assetUrl"
+              value={assetUrl}
+              onChange={(e) => setAssetUrl(e.target.value)}
+              placeholder="https://common---ForBiggerBlazes.mp4"
```

Comment:
Just a question, why this place holder and not `e.g. https://example.com/play.mp4` ?